surfsense_backend/alembic/versions/10_update_chattype_enum_to_qna_report_structure.py

```
@@ -0,0 +1,108 @@
+    
+    # Step 1: Rename the current enum type
+    op.execute(f"ALTER TYPE {CHAT_TYPE_ENUM} RENAME TO {old_enum_name}")
+    
+    # Step 2: Create the new enum type with new values
+    op.execute(f"CREATE TYPE {CHAT_TYPE_ENUM} AS ENUM({new_values_sql})")
+    
+    # Step 3: Add a temporary column with the new type
+    op.execute(f"ALTER TABLE {table_name} ADD COLUMN {column_name}_new {CHAT_TYPE_ENUM}")
+    
+    # Step 4: Update the temporary column with mapped values
+    op.execute(f"UPDATE {table_name} SET {column_name}_new = 'QNA' WHERE {column_name}::text = 'GENERAL'")
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)